app/src/main/res/values/strings.xml

```diff
@@ -1,58 +1,58 @@
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
 
-    <string name="app_name">Seedvault</string>
+    <string name="app_name" translatable="false">Seedvault</string>
     <string name="backup">Backup</string>
 
     <string name="restore_backup_button">Restore backup</string>
 
     <!-- Settings -->
     <string name="settings_backup">Backup my data</string>
-    <string name="settings_backup_location">Backup location</string>
-    <string name="settings_backup_location_picker">Choose backup location</string>
-    <string name="settings_backup_location_title">Backup Location</string>
-    <string name="settings_backup_location_invalid">The chosen location can not be used.</string>
+    <string name="settings_backup_location">Backup folder</string>
+    <string name="settings_backup_location_picker">Choose where to save the backup</string>
+    <string name="settings_backup_location_title">Backup folder</string>
+    <string name="settings_backup_location_invalid">The chosen folder can not be used.</string>
     <string name="settings_backup_location_none">None</string>
-    <string name="settings_backup_location_internal">Internal Storage</string>
+    <string name="settings_backup_location_internal">Internal storage</string>
     <string name="settings_backup_last_backup_never">Never</string>
-    <string name="settings_info">All backups are encrypted on your phone. To restore from backup you will need your 12-word recovery code.</string>
+    <string name="settings_info">All backups are encrypted by your device. To restore one you need your 12-word recovery code.</string>
```

Comment:
I would keep on device, given that they're encrypted on the device itself.

Also perhaps "to restore it" instead of "one"

Reply:
That is why I put in "by"… Otherwise it could strictly be that they were in the state of encrypted, on the device. Since the backups are not marked in sentence 1, and unreferenced "one" in sentence 2 fits.